### PR TITLE
Prevent hidden wizard steps to be focusable

### LINKED ISSUE #13127: Hidden wizard steps are focusable
### Package

filament/filament

### Package Version

v3.2.85

### Laravel Version

10.48.12

### Livewire Version

v3.5.0

### PHP Version

8.1.26

### Problem description

When using the Filament Wizard in accessibility mode or when navigating with the keyboard (tabbing), the form fields on the first step tab into the hidden fields of the second step instead of moving to the "Next" button. This behavior disrupts the form-filling process and impacts the accessibility and usability of the wizard.

Note: This issue does not occur in version v3.2.63; the tabbing functionality works as expected in that version.

### Expected behavior

When filling out the forms in the first step of the wizard, tabbing should sequentially move through all the form fields and then to the "Next" button. Only after clicking the "Next" button should the focus move to the fields in the second step.

### Steps to reproduce

1. Activate accessibility mode or navigate using the keyboard (tab key).
2. Fill out any form fields in the first step of the Filament Wizard.
3. Observe that tabbing skips to the hidden fields in the second step instead of the "Next" button.
4. Notice that the second step fields are tab-accessible even though they are not visible.

image

### Reproduction repository

https://github.com/izzudin96/filament-a11y

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by sebapastore**
## Description

Fixes #13127 

Recently, in #12985, the CSS invisibility strategy was changed to fix a UI  issue #9426 

However, removing the `invisible` class caused hidden steps to become focusable. In this PR, the `invisible` class is added back to resolve this issue. Adding it back does not affect the UI issue that was previously resolved.

- [X] Changes have been tested to not break existing functionality.

### CODE REVIEW

**Comment by izzudin96** [comment]
Thanks @sebapastore :)

File: packages/support/resources/views/components/section/index.blade.php [hunk 1]
```diff
@@ -172,7 +172,9 @@
             @if ($collapsed || $persistCollapsed)
                 x-cloak
             @endif
-            x-bind:class="{ 'absolute h-0 overflow-hidden border-none': isCollapsed }"
+            x-bind:class="{
+                'absolute invisible h-0 overflow-hidden border-none': isCollapsed,
```
Comment (danharrin) [inline packages/support/resources/views/components/section/index.blade.php]:
```suggestion
                'invisible absolute h-0 overflow-hidden border-none': isCollapsed,
```

**Comment by danharrin** [comment]
Thank you

**Comment by zepfietje** [comment]
Perfect fix, thanks @sebapastore!

**Comment by sebapastore** [comment]
Sorry for adding the bug. I will take this into account for future contributions!

**Pull request merged**